internal/build/manager.go

```diff
@@ -19,9 +21,41 @@ type Result struct {
 	Status  Status
 }
 
+// SyncOptions configures optional arguments of the Sync call.
+type SyncOptions struct {
+	// ContainerImage configures the image to be pushed. When left
+	// empty no image is pushed.
+	ContainerImage string
```

Comment:
WDYT regarding changing this field to boolean, and renaming it to something like pushImage? we have container imahge in the kmm mapping and in mod.Spec, and it might be confusing

Reply:
A boolean is what we had before (`pushImage`). Since image pushing is optional, I think it makes more sense to make the target image name an optional option.

Reply:
ok